src/Libraries/CoreNodesUI/UI/DoubleSliderNodeViewCustomization.cs

```diff
@@ -17,111 +9,16 @@ public class DoubleSliderNodeViewCustomization : INodeViewCustomization<DoubleSl
-            };
-
-            textBoxExpanderGrid.ColumnDefinitions.Add(new ColumnDefinition());
-            textBoxExpanderGrid.ColumnDefinitions.Add(new ColumnDefinition()
-            {
-                Width = new GridLength(29)
-            });
-
-            // input value textbox
-            var valtb = new DynamoTextBox(serializedValue);
-
-            Grid.SetColumn(valtb, 0);
-            textBoxExpanderGrid.Children.Add(valtb);
-
-            var exp = new Expander();
-            exp.Padding = new System.Windows.Thickness(4, 0, 0, 0);
-            Grid.SetColumn(exp, 1);
-            textBoxExpanderGrid.Children.Add(exp);
-
-            var sliderSp = new StackPanel()
-            {
-                HorizontalAlignment = HorizontalAlignment.Stretch
-            };
-            sliderSp.Children.Add(textBoxExpanderGrid);
-
-            // min max control
-            var minMaxControl = sliderSettingsControl;
-            minMaxControl.Visibility = Visibility.Collapsed;
-            sliderSp.Children.Add(minMaxControl);
-
-            // expander modifies visibility of min/max
-            exp.Expanded += (sender, args) =>
+            var slider = new UI.Controls.DynamoSlider(model, nodeView)
             {
-                minMaxControl.Visibility = Visibility.Visible;
+                DataContext = new SliderViewModel<double>(model)
             };
 
-            exp.Collapsed += (sender, args) =>
-            {
-                minMaxControl.Visibility = Visibility.Collapsed;
-            };
-
-            nodeUI.inputGrid.Children.Add(tbSlider);
-            nodeUI.PresentationGrid.Children.Add(sliderSp);
-            nodeUI.PresentationGrid.Visibility = Visibility.Visible;
-
-            tbSlider.DataContext = nodeModel;
-            valtb.DataContext = nodeModel;
-
-            // value input
-            valtb.BindToProperty(
-                new Binding("Value") { Mode = BindingMode.TwoWay, Converter = numberDisplayConverter });
-
-            // slider value 
-            var sliderBinding = new Binding("Value") { Mode = BindingMode.TwoWay, Source = nodeModel };
-            tbSlider.SetBinding(RangeBase.ValueProperty, sliderBinding);
-
-            // max slider value
-            var bindingMaxSlider = new Binding("Max")
-            {
-                Mode = BindingMode.OneWay,
-                Source = nodeModel,
-                UpdateSourceTrigger = UpdateSourceTrigger.Explicit
-            };
-            tbSlider.SetBinding(RangeBase.MaximumProperty, bindingMaxSlider);
+            nodeView.inputGrid.Children.Add(slider);
+        }
 
-            // min slider value
-            var bindingMinSlider = new Binding("Min")
-            {
-                Mode = BindingMode.OneWay,
-                Source = nodeModel,
-                UpdateSourceTrigger = UpdateSourceTrigger.Explicit
-            };
-            tbSlider.SetBinding(RangeBase.MinimumProperty, bindingMinSlider);
+        public void Dispose()
```

Comment:
Is this necessary by transitive interface requirements? If not, can we remove it?